lib/rialto/etl/writers/sparql_statement_writer.rb

```diff
@@ -27,42 +26,44 @@ def serialize(context)
         end
 
         # rubocop:disable Metrics/CyclomaticComplexity
+        # rubocop:disable Metrics/PerceivedComplexity
         def serialize_hash(hash, graph_name = nil)
           statements = []
-          subject_id = hash['@id'].to_s
-          subject = RDF::URI.new(hash['@id_ns'] + subject_id)
+          subject = RDF::URI.new(hash['@id'])
           graph_name ||= hash['@graph']
 
-          # Always add a valid date
-          statements << values_to_delete_insert(subject, Vocabs::DCTERMS['valid'], Time.now.to_date, graph_name, true)
-
           # Type
-          statements << values_to_delete_insert(subject, RDF.type, hash['@type'], graph_name, hash.key?('!type'))
-
-          # Label
-          # SKOS.prefLabel & VCARD.fn
-          statements << values_to_delete_insert(subject,
-                                                [Vocabs::SKOS['prefLabel'],
-                                                 Vocabs::FOAF['fn']], hash['@label'],
-                                                graph_name,
-                                                hash.key?('!label'))
-
-          # Person name vcard
-          statements << person_name_to_statements(subject, hash, graph_name) if hash.key?('@person_name')
-
-          # Person address vcard
-          statements << person_address_to_statements(subject, hash, graph_name) if hash.key?('@person_address')
+          statements << values_to_delete_insert(subject, RDF.type, hash['@type'], graph_name, true) if hash.key?('@type')
 
-          # Advisees
-          statements << advisees_to_statements(subject, subject_id, hash, graph_name) if hash.key?('@advisees') &&
-                                                                                         !hash['@advisees'].empty?
+          hash.each do |field, value|
+            # Ignore any @fields or !fields or #fields
+            next if field.start_with?('@', '!', '#')
+            values = value.is_a?(Array) ? value : [value]
```

Comment:
How would feel about simplifying this line to `values = Array(value)` which should have the same effect?

Reply:
They're different for hashes:
```
irb(main):001:0> Array({'a' => 1})
=> [["a", 1]]
irb(main):002:0> x = {'a' => 1}
=> {"a"=>1}
irb(main):003:0> Array(x)
=> [["a", 1]]
irb(main):004:0> [x]
=> [{"a"=>1}]
```

Unit tests fail if switch because of this difference.